File: Java/app/src/main/java/dev/ogabek/java/helper/CallActivities.java
package dev.ogabek.java.helper;

import android.content.Context;
import android.content.Intent;

import dev.ogabek.java.activity.ProfileActivity;
import dev.ogabek.java.model.Member;

public class CallActivities {

    public Intent openProfileActivityWithData(Context context, Member member) {
        Intent intent = new Intent(context, ProfileActivity.class);
        intent.putExtra("member", member);
        return intent;
    }

}

File: Kotlin/app/src/main/java/dev/ogabek/kotlin/helper/CallActivities.kt
package dev.ogabek.kotlin.helper

import android.content.Context
import android.content.Intent
import dev.ogabek.kotlin.activity.ProfileActivity
import dev.ogabek.kotlin.model.Member

class CallActivities {

    fun openProfileActivityWithData(context: Context, member: Member): Intent {
        val intent = Intent(context, ProfileActivity::class.java)
        intent.putExtra("member", member)
        return intent
    }

}